packages/calling/calling/src/CallingClient/CallingClient.ts

```diff
@@ -518,6 +163,10 @@ export class CallingClient extends Eventing<CallingClientEventTypes> implements
   public makeCall = (dest: CallDetails): ICall | undefined => {
     let call;
 
+    // this is a temporary logic to get registration obj
+    // it will change once we have proper lineId and multiple lines as well
+    const {registration} = Object.values(this.lineDict)[0];
```

Comment:
this will change once you move to map

Reply:
yeah this is a temporary hack as we don't have proper lineId through which we can fetch it.